src/clr_bundle_rm.c

```diff
@@ -46,7 +46,10 @@ static void print_help(const char *name)
 	printf("   -h, --help              Show help options\n");
 	printf("   -p, --path=[PATH...]    Use [PATH...] as the path to verify (eg: a chroot or btrfs subvol\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
```

Comment:
bundle-remove needs both version and content URLs, so the -u option is available for convenience.